src/vma/sock/sockinfo_udp.cpp

```diff
@@ -2760,6 +2761,42 @@ size_t sockinfo_udp::handle_msg_trunc(size_t total_rx, size_t payload_size, int
 	return total_rx;
 }
 
+int sockinfo_udp::get_socket_tx_ring_fd(struct sockaddr *to, socklen_t tolen)
+{
+	NOT_IN_USE(tolen);
+	si_udp_logfunc("get_socket_tx_ring_fd fd %d to %p tolen %d", m_fd, to ,tolen);
+
```

Comment:
Why does "tolen" parameter exist if we don't use it at all ?

Reply:
always when passing address it's len is passed, this keeps some alignment with other socket based calls (connect bind).
i can add (maybe should) add a test to verify the size